src/server/stream_family.cc

```diff
@@ -817,6 +927,46 @@ OpStatus OpDestroyGroup(const OpArgs& op_args, string_view key, string_view gnam
   return OpStatus::SKIPPED;
 }
 
+struct GroupConsumerPair {
+  streamCG* group;
+  streamConsumer* consumer;
+};
+
+struct GroupConsumerPairOpts {
+  string_view group;
+  string_view consumer;
+};
+
+OpResult<vector<GroupConsumerPair>> OpGetGroupConsumerPairs(ArgSlice slice_args, OpArgs op_args,
```

Comment:
I do not see that this function returns any status besides the value. If it can not fail, it's better remove `OpResult` wrapping.

Reply:
Ok, sure